runtime/autoload/provider/ruby.vim

```diff
@@ -16,23 +16,27 @@ function! s:job_opts.on_stderr(chan_id, data, event)
 endfunction
 
 function! provider#ruby#Detect() abort
-  return exepath('neovim-ruby-host')
+  if exists("g:ruby_host_prog")
+    return g:ruby_host_prog
+  else
+    return exepath('neovim-ruby-host')
+  end
 endfunction
 
 function! provider#ruby#Prog()
   return s:prog
 endfunction
 
 function! provider#ruby#Require(host) abort
-  let args = [provider#ruby#Prog()]
+  let prog = provider#ruby#Prog()
   let ruby_plugins = remote#host#PluginsForHost(a:host.name)
 
   for plugin in ruby_plugins
-    call add(args, plugin.path)
+    let prog .= " " . shellescape(plugin.path)
```

Comment:
I want to support arbitrary commands for this, not just executables, and my understanding was calling `jobstart` with a string rather than a list gives you e.g. `bash -c '...'` execution. Is there a better way to support this?